src/components/Navbar/NavBar.tsx

```diff
@@ -47,17 +47,17 @@ const NavBar = (props: Props) => {
 
   const dropdownOptions = [
     { url: '/merchants', translation: 'donate', external: false },
+    {
+      url: '/light-up-chinatown',
+      translation: 'light-up-chinatown',
+      external: false,
+    },
```

Comment:
I don't think this is supposed to be a part of the dropdown, but is supposed to always be showing to the right of Ways to Donate

![image](https://user-images.githubusercontent.com/4944733/99219376-571a1380-2791-11eb-9f68-2abf5e22bed6.png)